ddtrace/appsec/_common_module_patches.py

```diff
@@ -33,7 +33,7 @@
 _RASP_POPEN = "rasp_Popen"
 
 
-def patch_common_modules():
+def patch_common_modules(testing=False):
```

Comment:
would it be possible to avoid the use of a [boolean flag](https://adamj.eu/tech/2021/07/10/python-type-hints-how-to-avoid-the-boolean-trap/) here?